chain-impl-mockchain/src/evm/mod.rs

```diff
@@ -283,4 +353,12 @@ mod test {
             }
         }
     }
+
+    quickcheck! {
+        fn evm_transaction_serialization_bijection(b: EvmTransaction) -> bool {
```

Comment:
Good, we should also add `EvmTransaction` to `Fragment::arbitrary` as it would have uncovered the error above